src/test/scala/org/clulab/asist/text/TestCommunications.scala

```diff
@@ -0,0 +1,34 @@
+package org.clulab.asist.text
+
+import org.clulab.asist.BaseTest
+// For testing team communications
+
+class TestCommunications extends BaseTest {
+
+ // behavior of "team_communication.yml"
+
+
+  passingTest should "Parse existential constructions" in {
+    val text = "There is a medkit in the library. "
+
+    val mentions = extractor.extractFromText(text)
+    val medkit = DesiredMention("MedKit", "medkit")
+    val location = DesiredMention("Infrastructure", "library")
+    val ex1 = DesiredMention("KnowledgeSharing", "There is a medkit in the library", Map("location" -> Seq(location), "exists" -> Seq(medkit)))
+
+    testMention(mentions, ex1)
+
+  }
+
+  failingTest should "Parse existential constructions 2" in {
+    val text =  "There is a victim in here."
+    // fixme:{I dont know why this one fails! Help}
```

Comment:
can you make an issue from this and assign it to me, please?
in the issue you can reference the "future line 25 of TestCommunications" or something since it's not merged yet.
Otherwise I think we'll lose track of it, thanks!

Reply:
Yes, will do

Reply:
thanks!